common/script/ops/addWebhook.js

```diff
@@ -10,7 +10,7 @@ module.exports = function addWebhook (user, req = {}) {
   let wh = user.preferences.webhooks;
 
   if (!validator.isURL(_.get(req, 'body.url'))) throw new BadRequest(i18n.t('invalidUrl', req.language));
-  if (!validator.isBoolean(_.get(req, 'body.enabled'))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
+  if (!validator.isBoolean(String(_.get(req, 'body.enabled')))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
```

Comment:
Yeah I tried to upgrade to version 5 but this blocked me. You cannot even pass `undefined` or `null` as it refuses to validate them...